exporter/internal/queue/default_batcher_test.go

```diff
@@ -288,18 +304,22 @@ func TestDefaultBatcher_Shutdown(t *testing.T) {
 	batchCfg.MinSizeItems = 10
 	batchCfg.FlushTimeout = 100 * time.Second
 
+	ba, err := NewBatcher(batchCfg,
+		func(ctx context.Context, req internal.Request) error { return req.Export(ctx) },
+		2)
+	require.NoError(t, err)
+
+	// TODO: https://github.com/open-telemetry/opentelemetry-collector/issues/12244
```

Comment:
Should we revert https://github.com/open-telemetry/opentelemetry-collector/pull/12122 until this is fixed?

Reply:
I was thinking the same.

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/12246